src/util/link-helper.ts

```diff
@@ -63,3 +63,27 @@ function isRelativeLink(input: string) {
 function hasRelativeProtocol(input: string) {
     return input.startsWith('//');
 }
+
+/**
+ * Returns the appropriate `rel` attribute value for a link.
+ *
+ * If an explicit `rel` value is provided, it will be used.
+ * Otherwise, when `target` is `_blank`, automatically returns
+ * `"noopener noreferrer"` for improved security.
+ *
+ * @param target - The target attribute value (e.g., "_blank", "_self")
+ * @param explicitRel - An explicitly provided rel attribute value
+ * @returns The rel attribute value to use, or undefined if none needed
+ */
+export function getRel(target?: string, explicitRel?: string) {
```

Comment:
It feels like it ought to be very easy for an AI to whip up a nice suite of tests for this helper.

Reply:
That sucks to hear 😢

I'll add tests, then people can come complain to me 😂